src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -311,6 +313,7 @@ function bindFeedbackSessionEditFormSubmission() {
                 }
 
                 if (!hasError) {
+                    removeLoadingIndicator($('#button_submit'), 'Save Changes');
```

Comment:
The spinner should be removed even when there's an error returned

Reply:
You are right, of course. I fixed that.